executor/show.go

```diff
@@ -166,11 +166,30 @@ func (e *ShowExec) fetchShowEngines() error {
 	return nil
 }
 
+// move information_schema to the first, and the other are sorted in the origin ascending order.
+func moveInfoSchemaToFront(dbs []string) {
+	if strings.EqualFold(dbs[0], "INFORMATION_SCHEMA") {
+		return
+	}
+
+	for i := 1; i < len(dbs); i++ {
+		if strings.EqualFold(dbs[i], "INFORMATION_SCHEMA") {
+			tmp := dbs[i-1]
```

Comment:
this line can be removed.

Reply:
I want to replace to copy(dbs[1:i+1], dbs[0:i]) before. but the i+1 in the dbs[1:i+1] is not in the work Interval. so I changed  to the code in the file.